### DISCUSSION TITLE
Incomplete elapsed time display

### DISCUSSION THREAD

**Discussion by etimal04180** [rern/rAudio#749] [Fixed Issues]
Hi,
installed player last week, all working fine and very satisfactory but I'm very surprised at the way elapsed time is displayed during the first 59 seconds without any mention of minutes. All the time display screens I've seen show 0:12 or 0:48 (for example) not 12 or 48.
Is this a bug you intend to rectify or a deliberate choice (in that case, maybe I'm not the only one surprised).
Thanks for all the work, good to see this project is still alive and kicking !

**Comment by EarsofTin** [comment]
I don't  think it's a bug - since it changes as you watch no confusion with minutes. I like the cleaner, leaner appearance.  rAudio has the best UI in terms of look and feel IMO.  I also appreciate the the time counter on the queue page since that's the page I look at the most (I keep buttons permanently enabled).

**Comment by gitso7** [comment]
I'd prefer the format 0:12, too.
Maybe it can be made configurable (to satisfy both "needs")
